logger/mock.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package logger
+
+var _ Logger = (*loggerMock)(nil)
+
+type loggerMock struct{}
+
+// NewMock returns wrapped go kit logger mock.
+func NewMock() (Logger, error) {
+	return &loggerMock{}, nil
+}
```

Comment:
No need to return an error here, since it's always `nil`. It will also simplify using the mock (instead of `logger, _ := logger.NewMock()` simply call `logger.NewMock()`)